server/queues/processors/NotificationsProcessor.ts

```diff
@@ -230,18 +230,28 @@ export default class NotificationsProcessor extends BaseProcessor {
         const user = await User.findByPk(collaboratorId, { transaction });
 
         if (user) {
-          // `user` has to have `createSubscription` permission on `document`.
-          if (can(user, "createSubscription", document)) {
+          // `user` has to have `subscribe` permission on `document`.
+          if (can(user, "subscribe", document)) {
             // `subscriptionCreator` uses `findOrCreate`.
             // A duplicate won't be created if a subscription
             // exists already.
-            await subscriptionCreator({
-              user: user,
-              documentId: document.id,
-              event: "documents.update",
+            const exists = await Subscription.findOne({
+              where: { userId: user.id, documentId: document.id },
```

Comment:
```suggestion
              where: { userId: user.id, documentId: document.id, event: "documents.update" },
```

`event` should be in this query, no?

Reply:
Indeed should